src/controllers/accommodation.controller.js

```diff
@@ -42,6 +43,44 @@ class AccommodationController {
     ResponseService.setSuccess(200, 'list of available accommodations', availableAccommodations);
     return ResponseService.send(res);
   }
+
+  /**
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} created Accomodation
+   * @memberof AccommodationController
+   */
+  static async createAccomodation(req, res) {
+    const { name, typeId, locationId, accommodationPictures, roomList } = req.body;
+    const accomodation = await AccommodationService
+      .createAccomodation({ name, typeId, locationId });
+
+    // create accommodation pictures
+    const pictures = await ImageService.createImage({ subjectId: accomodation.get().id,
+      subjectType: 'accommodation',
+      imageUrl: accommodationPictures });
+
+    const createRoom = async (room) => {
+      const { dataValues } = await AccommodationService.createRoom(room);
```

Comment:
Why did you assign createRoom to dataValues variable while you didn't use that dataValues anywhere? if you remove it, it can't work?

Reply:
line 66 I am getting the id in datavalue